helm/Chart.yaml

```diff
@@ -27,3 +27,12 @@ dependencies:
   - name: nats
     version: "1.3.7"
     repository: "https://nats-io.github.io/k8s/helm/charts/"
+
+annotations:
+  catalog.cattle.io/auto-install: SBOMbastic=0.1.0-alpha1
+  catalog.cattle.io/display-name: SBOMbastic
+  catalog.cattle.io/namespace: sbombastic
+  catalog.cattle.io/release-name: rancher-sbombastic
+  catalog.cattle.io/requests-cpu: "500m"
+  catalog.cattle.io/requests-memory: "128Mi"
```

Comment:
We don't have any value inside of the helm chart, how did you come up with these numbers?

@fabriziosestito do you think they make sense?